crates/routes/src/images.rs

```diff
@@ -66,7 +66,9 @@ async fn upload(
     .await
     .map_err(error::ErrorBadRequest)?;
 
-  let images = res.json::<Images>().await.map_err(error::ErrorBadRequest)?;
+  // Note: res.json() is currently broken.
```

Comment:
Could you include an issue link?

Reply:
I'll try to provide the error and link one.

Reply:
Nevermind, this one was a non-issue. The gzip was the only problem.